src/User/composer.json

```diff
@@ -22,6 +22,7 @@
     },
     "require-dev": {
         "doctrine/orm": "^2.6",
+        "lexik/jwt-authentication-bundle": "dev-master",
```

Comment:
can you put it in alpha order. Instead of dev-master use `^2.4`

Reply:
I'm using Lexik\Bundle\JWTAuthenticationBundle\Security\User\UserProviderWithPayloadSupportsInterface that it is not in the ^2.4 branch so if you don't want to have a dev branch dependency, we will have to wait the next release from JWT bundle.

Reply:
i see. Worth waiting? not sure when they expect this to be released, otherwise dev-master is fine.